themes/root/templates/searchapi/openapi.phtml

```diff
@@ -216,6 +225,16 @@
                             }
                         }
                     },
+                    "200 (resumption token)": {
```

Comment:
I don't think we need a separate response type for resumption token. Just add it to the normal response as optional.

Reply:
Alrighty!